libnetwork_test.go

```diff
@@ -97,7 +97,7 @@ func getPortMapping() []types.PortBinding {
 }
 
 func TestNull(t *testing.T) {
-	network, err := createTestNetwork("null", "testnetwork", options.Generic{})
+	network, err := createTestNetwork("null", "testnull", options.Generic{})
```

Comment:
So I found the reason. There is a bug in bridge driver: On Network deletion we do not cleanup the respective configuration. So now the sanity checks detect a conflict when a new network gets created and has same id of a previous existing one.